javascript/script.js

```diff
@@ -0,0 +1,22 @@
+//script.js
+
+var a = -2;
+var b = -8;
+var value = (a * a) + (2 * a * b) - (b * b);
+console.log('Wynik to ' + value);
+
+if (value > 0) {
+	console.log('Wartość jest dodatnia');
+} else if (value < 0) {
+	console.log('Wartość jest ujemna');
+} else {
+	console.log('Wartość zerowa');
+}
+
+if (value == 0) {
+	console.log('Wartość zerowa');
+} else /*if -niepotrzebny*/ (value != 0) {
```

Comment:
warunek jest też niepotrzebny - on jest częścią składową `if`a

Nie potrzebujesz sprawdzać czy `value != 0` bo Ty to na tym etapie wiesz